roles/lmod/tasks/main.yml

```diff
@@ -73,46 +79,52 @@
     dest: /etc/profile.d
     owner: root
     group: root
-    mode: 0777
+    mode: 0755
   tags:
     - configuration
 
 # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=891541
 - name: "bugfix for lmod posix_c"
   become: yes
   register: lmod_bugfix_posix
-  shell: |
-    ln -s -f /usr/lib/x86_64-linux-gnu/lua/5.1/posix_c.so /usr/lib/x86_64-linux-gnu/lua/5.1/posix.so
-    ln -s -f /usr/lib/x86_64-linux-gnu/lua/5.2/posix_c.so /usr/lib/x86_64-linux-gnu/lua/5.2/posix.so
-    ln -s -f /usr/lib/x86_64-linux-gnu/lua/5.3/posix_c.so /usr/lib/x86_64-linux-gnu/lua/5.3/posix.so
+  file:
+    src: "/usr/lib/x86_64-linux-gnu/lua/{{ item }}/posix_c.so"
+    dest: "/usr/lib/x86_64-linux-gnu/lua/{{ item }}/posix.so"
+    owner: "root"
+    group: "root"
+    state: "link"
+    force: true
   when: 
     - ansible_distribution == 'Ubuntu'
     - ansible_distribution_version == '18.04'
-  changed_when: lmod_bugfix_posix.rc != 0
```

Comment:
Is it okay that we dropped this?

Reply:
Yes - this needed to be dropped because we aren't executing commands anymore, we are using the built-in file module. So there isn't even a `lmod_bugfix_posix.rc` variable to check anymore!

Reply:
Makes sense!